src/fugw/mappings/dense.py

```diff
@@ -279,6 +279,10 @@ def transform(self, source_features, device="auto"):
         ----------
         source_features: ndarray(n_samples, n) or ndarray(n)
             Contrast map for source subject
+        id_reg: float, in the [0, 1] interval, defaults to 0
+            If source/target share the same geometry,
+            interpolate the transport plan with the identity
+            using the provided coefficient.
```

Comment:
Maybe add a description of what each value corresponds to:
```suggestion
            using the provided coefficient.
            A value of 1 (resp. 0) will rely solely on the identity
            (resp. the transport plan).
```